Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -349,6 +349,13 @@ private static void ReadDurableConfig(string hostConfigPath, out Dictionary<stri
             var json = JObject.Parse(File.ReadAllText(hostConfigPath));
             JToken durableTaskValue;
 
+            // For Functions V2: the 'durableTask' property is set under the 'extensions' property.
+            JToken extensionsValue;
+            if (json.TryGetValue(Constants.Extensions, StringComparison.OrdinalIgnoreCase, out extensionsValue) && extensionsValue != null)
+            {
+                json = (JObject)extensionsValue;
+            }
+
```

Comment:
In corner case if both exists, we will honor V2 format.  Regression paranoid speaking, perhaps we could move this check after the V1 check was not found.

Reply:
Note that v2 runtime blows up if you have it the old way. Not sure if v1 blows up if you have it the new way. I think this is too much of an edge case to be worth changing. If they have both in there, they probably have bigger problems.

Reply:
I just tested configuring in both locations for a V2 function app (using the latest CLI) and can confirm that the runtime **did not** blow up. I'll note this as a workaround.

Reply:
@cgillum hopefully we can have the fixed deployed soon enough that we don't need to spread this workaround too much.